### ISSUE TITLE
Extend Controls with Keyboard Shortcuts

### ISSUE THREAD

**Issue by madhaven** [madhaven/proTracker#41]
Have keyboard shortcuts to control the UI.
Expected to be intuitive.

- [ ] ~study existing shortcut practices~
- [x] Menu Screen Shortcut
- [x] Menu Shortcut to each Menu Item
- [x] New Entry shortcut
- [x] ~Task Status Change Shortcut / click shortcut on logs page~
- [x] ~Task Status Change Shortcut / click shortcut on projects page~

Lower Priority updates

* fetch shortcuts from saved preferences
  #56 
* navigate with keyboard: waiting on advanced menu implementation for items.

Thanks @kurieneapen

**Comment by madhaven** [comment]
https://www.electronjs.org/docs/latest/tutorial/keyboard-shortcuts
